airmozilla/manage/archiver.py

```diff
@@ -23,10 +23,13 @@ def auto_archive():
                 template__name__contains='Vid.ly')
     )
     for event in events:
-        archive(event, swallow_email_exceptions=True)
+        archive(event, swallow_email_exceptions=True, verbose=verbose)
 
+        if event == events[-1] and verbose:  # pragma: no cover
```

Comment:
Did you try this? I don't think it works. 

`events` is not a list. It's a QuerySet object that you can iterate on like a list. 
So something like this:
```python
any = False
for event in events:
    archive(event, ...)
    any = True
if any:
    ...
```